frontend/src/metabase-types/api/permissions.ts

```diff
@@ -19,6 +19,7 @@ export type GroupPermissions = {
 export type DownloadPermission = "full" | "limited" | "none";
 
 export type DownloadAccessPermission = {
+  native?: DownloadSchemasPermission;
```

Comment:
I changed these because of the responses I was receiving from the API that were related to these types i.e. the permissions for a local database could look something like this:

```
{
    "data": {
        "native": "write",
        "schemas": "all"
    },
    "download": {
        "native": "full",
        "schemas": "full"
    },
    "data-model": {
        "schemas": "all"
    },
    "details": "yes"
}
```

while another could look like this:

```
{
    "download": {
        "native": "full",
        "schemas": "full"
    },
    "data": {
        "native": "write",
        "schemas": "all"
    }
}
```

Reply:
@noahmoss do you know or remember why we have `download: { native: "full" }` subtree?

Reply:
IIRC native download permissions are broken out for ease of enforcement on the backend. I don't think they don't strictly need to be in the tree since the backend updates it automatically based on the value of the main download perms.